subsys/mpsl/Kconfig

```
@@ -25,6 +25,107 @@ config MPSL_TIMESLOT_SESSION_COUNT
 	help
 	  Maximum number of timeslot sessions.
 
+DT_COMPAT_NORDIC_NRF21540_GPIO := nordic,nrf21540-fem
+DT_COMPAT_SKYWORKS_SKY66112_11 := skyworks,sky66112-11
+
+config MPSL_FEM_NRF21540_GPIO_SUPPORT
+	bool
+	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_NORDIC_NRF21540_GPIO)) && !BT
+
+config MPSL_FEM_SKY66112_11_SUPPORT
+	bool
+	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_SKYWORKS_SKY66112_11))
+
+config MPSL_FEM
+	bool "Radio front-end module (FEM) support"
+# MPSL FEM is not supported on 53 yet
+	depends on !(SOC_NRF5340_CPUAPP || SOC_NRF5340_CPUNET) && \
+	           (MPSL_FEM_NRF21540_GPIO_SUPPORT || MPSL_FEM_SKY66112_11_SUPPORT)
```

```suggestion
	           (MPSL_FEM_NRF21540_GPIO_SUPPORT || MPSL_FEM_SKY66112_11_SUPPORT)
	default y
```
It seems default is still missing

---

:thumbsup: 
I took this into account when I introduced a separate file "Kconfig.fem"